blockmanager.go

```diff
@@ -411,7 +420,6 @@ func (b *blockManager) cfHandler() {
 	// done.
 	defer func() {
```

Comment:
function closure not needed anymore

Reply:
I removed the outdated comment.
I thought of leaving the rest for leaving the  logging behavior as it was. Do you suggest to remove that?

Reply:
Oh, is just meant that this cane be `defer log.Trace("Committed filter header handler done")` now :)

Reply:
sure thing! fixed.